packages/fx-core/resource/strings.json

```diff
@@ -23,7 +23,7 @@
     "AlreadyProvisionNotice": "'%s' is already provisioned in the cloud, will create the teams app again if it has been removed.",
     "NotProvisionedNotice": "'%s' is not provisioned in the cloud yet. Please provision first.",
     "GenerateArmTemplateFailNotice": "'%s' Failed to create ARM template for required Azure services.",
-    "RegenerateArmTemplateConfirmNotice": "We will update your Azure resource template based on your resource and capability selection. The existing template file will be moved to backup folder under Azure resource template folder for your reference. We recommend to initialize your project workspace with git init to track all changes.",
+    "RegenerateArmTemplateConfirmNotice": "We will update your Azure resource template based on your resource and capability selection. We recommend to initialize your project workspace with git init to track all changes.",
```

Comment:
Maybe change to: `The Azure resource template is updated based on your resource and capability selection.....`

Reply:
sure, thanks!